### ISSUE TITLE
Enhancement: add “no license” or “unknown license” to glossary

### ISSUE THREAD

**Issue by effjot** [3liz/qgis-pgmetadata-plugin#159]
Similar to the “unknown“ publication frequency, it would be useful to document if there is no license and/or license terms are unknown and if confidentiality is unknown.

Suggested addition to glossary:
```
field = 'dataset.license'
code = 'NO'
label_en = 'No or unknown license'
description_en = 'Dataset has been published explicitly without license or no license conditions have been documented'

field = 'dataset.confidentiality'
code = 'UNK'
label_en = 'Unknown'
description_en = 'Access restrictions for this dataset are unknown'
```

The cases of explictly no license and unknown licenses could be separated, if there is an use case for that.

I could prepare a PR if you like.

**Comment by Gustry** [comment]
It's indeed different than a "NULL" value in the DB.
It looks fine for me.

**Comment by effjot** [comment]
I have just realised that unkown publication frequency was a local addition and is not in the main branch. ;-)

So maybe add that as well:

```
field = 'dataset.publication_frequency'
code = 'UNK'
label_en = 'Unknown'
description_en = 'Frequency of maintenance for the data is not known'
```

**Comment by effjot** [comment]
Fixed by PR #103
